encoding/decode.go

```diff
@@ -0,0 +1,33 @@
+package encoding
+
+import (
+	"bufio"
+	"bytes"
+
+	cbor "github.com/polydawn/refmt/cbor"
+	"github.com/polydawn/refmt/pretty"
+	"github.com/polydawn/refmt/shared"
+)
+
+//DecodeCBOR returns a string representation of a CBOR blob
+func DecodeCBOR(blob []byte) (string, error) {
```

Comment:
I think the name of this function could be better.
It isn't obvious from the name what this function does.

I think `HumanReadable` might be a better name, do you agree?

Reply:
Fine by me, I used Decode to try and keep with the refmt naming system. I could also move it into the main package namespace instead of the encoding namespace. Thoughts?

Reply:
With name change, SGWM.

Reply:
Do you want a filename change to go with the function name change?

Reply:
Why not.